<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Software â€¢ The Mills Lab</title>
    
    
    
    	<!-- Twitter Cards -->
	<meta name="twitter:title" content="Software">
	
	
	
	
	<meta name="twitter:card" content="summary_large_image">
	<meta name="twitter:image" content="/images/banner-short-apb.png">
	
	<!-- Open Graph -->
	<meta property="og:locale" content="en">
	<meta property="og:type" content="article">
	<meta property="og:title" content="Software">
	
	<meta property="og:url" content="/software/">
	<meta property="og:site_name" content="The Mills Lab">

    <link rel="canonical" href="/software/">

    <link href="/atom.xml" type="application/atom+xml" rel="alternate" title="The Mills Lab Atom Feed">
    <link href="/sitemap.xml" type="application/xml" rel="sitemap" title="Sitemap">

    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="cleartype" content="on">

    <link rel="stylesheet" href="/css/main.css">
    <!-- HTML5 Shiv and Media Query Support for IE -->
    <!--[if lt IE 9]>
      <script src="/js/vendor/html5shiv.min.js"></script>
      <script src="/js/vendor/respond.min.js"></script>
    <![endif]-->

  </head>

  <body id="js-body">
    <!--[if lt IE 9]><div class="upgrade notice-warning"><strong>Your browser is quite old!</strong> Why not <a href="http://whatbrowser.org/">upgrade to a newer one</a> to better enjoy this site?</div><![endif]-->

    <header id="masthead">
  <div class="inner-wrap">
    <a href="/" class="site-title">The Mills Lab</a>
    <nav role="navigation" class="menu top-menu">
        <ul class="menu-item">
	<li class="home"><a href="/">The Mills Lab</a></li>
	
    
    <li><a href="/news/" >News</a></li>
  
    
    <li><a href="/research/" >Research</a></li>
  
    
    <li><a href="/people/" >People</a></li>
  
    
    <li><a href="/publications/" >Publications</a></li>
  
    
    <li><a href="/software/" >Software</a></li>
  
    
    <li><a href="/contact/" >Contact</a></li>
  
</ul>
    </nav>
  </div><!-- /.inner-wrap -->
</header><!-- /.masthead -->
    <nav role="navigation" id="js-menu" class="sliding-menu-content">
  <h5>The Mills Lab <span>Table of Contents</span></h5>
  <ul class="menu-item">
    <li>
      <a href="/news/">
        
        <div class="title">News</div>
        
      </a>
    </li><li>
      <a href="/research/">
        
        <div class="title">Research</div>
        
      </a>
    </li><li>
      <a href="/people/">
        
        <div class="title">People</div>
        
      </a>
    </li><li>
      <a href="/publications/">
        
        <div class="title">Publications</div>
        
      </a>
    </li><li>
      <a href="/software/">
        
        <div class="title">Software</div>
        
      </a>
    </li><li>
      <a href="/contact/">
        
        <div class="title">Contact</div>
        
      </a>
    </li>
  </ul>
</nav>
<button type="button" id="js-menu-trigger" class="sliding-menu-button lines-button x2" role="button" aria-label="Toggle Navigation">
  <span class="nav-lines"></span>
</button>

<div id="js-menu-screen" class="menu-screen"></div>


    <div id="page-wrapper">
      <div id="main" role="main">			
	<div class="wrap">
		
		<div class="page-feature">
			<div class="page-image">
				<img src="/images/banner-short-apb.png" class="page-feature-image" alt="Software">
				
			</div><!-- /.page-image -->
		</div><!-- /.page-feature -->
		
		<div class="page-title">
			<h1>Software</h1>
			
		</div>
		<div class="archive-wrap">
			<div class="page-content">
				
<p>

        <div class="Nheader"><b><a href="https://github.com/mills-lab/spectre">spectre</a></b></div>
<p>This software is designed to identify regions of active translation from ribsome profiling sequence data. This analytical pipeline scores the translational status of each annotated region (5'UTR, CDS, exon, 3'UTR) as a function of its spectral coherence over user-defined N nucleotide sliding windows to an idealized reference coding signal.</p>
<p>Source code and documentation is available at <a href="https://github.com/mills-lab/spectre">https://github.com/mills-lab/spectre</a>.</p>
<p>Reference:<br />
Chun SY, Rodriguez CM, Todd PK, Mills RE. SPECtre&#58; a spectral coherence-based classifier of actively translated transcripts from ribosome profiling sequence data. bioRxiv, 2015, DOI 10.1101/034777 
</p>
<br />


        <div class="Nheader"><b><a href="https://github.com/mills-lab/svelter">svelter</a></b></div>
<p>This software is designed to identify both simple and complex rearrangements from paired-end sequencing data.</p>
<p>Source code and documentation is available at <a href="https://github.com/mills-lab/svelter">https://github.com/mills-lab/svelter</a>.</p>
<p>Reference:<br />
Zhao X, Emery SB, Myers B, Kidd JM, Mills RE. Resolving complex structural genomic rearrangements using a randomized approach. Genome Biol 2016, 17(1), 126 
</p>
<br />


        <div class="Nheader"><b><a href="https://github.com/mills-lab/dinumt/">dinumt</a></b></div>
<p>This software is designed to identify and genotype nuclear insertions of mitochondrial origin from whole genome sequence data. It consists of two programs, diumt (di-nu-mite), which identifies sites of insertions in a single sample and gnomit (geno-mite), which genotypes those sites across multiple samples.</p>
<p>Source code and documentation is available at <a href="https://github.com/mills-lab/dinumt/">https://github.com/mills-lab/dinumt/</a>.</p>
<p>Reference:<br />
Dayama G, Emery SB, Kidd JM, Mills RE. The genomic landscape of polymorphic human nuclear mitochondrial insertions. Nucleic Acids Res 2014 Nov 10, 42(20), 12640-9 
</p>
<br />


        <div class="Nheader"><b><a href="https://github.com/mills-lab/canny/">canny</a></b></div>
<p>This software was used to analyze 2,135 Agilent 1M CGH arrays designed to assess copy number variation genotypes in Phase 3 samples from the 1000 Genomes Project.</p>
<p>Source code and documentation is available at <a href="https://github.com/mills-lab/canny/">https://github.com/mills-lab/canny/</a>.</p>
<p>Reference:<br />
Sudmant PH, Rausch T, Gardner EJ, Handsaker RE, Abyzov A et al. An integrated map of structural variation in 2,504 human genomes. Nature 2015 Oct 1, 526(7571), 75-81 
</p>
<br />



</p>

			</div><!-- /.page-content -->
		</div class="archive-wrap"><!-- /.archive-wrap -->
	</div><!-- /.wrap -->
</div><!-- /#main -->
      <footer role="contentinfo" id="site-footer">
	<nav role="navigation" class="menu bottom-menu">
		<ul class="menu-item">
		
      
			<li><a href="http://www.med.umich.edu/" target="_blank">University of Michigan Medical School</a></li>
		
      
			<li><a href="https://medicine.umich.edu/dept/computational-medicine-bioinformatics" target="_blank">Department of Computational Medicine and Bioinformatics</a></li>
		
      
			<li><a href="https://medicine.umich.edu/dept/human-genetics/" target="_blank">Department of Human Genetics</a></li>
		
		</ul>
	</nav><!-- /.bottom-menu -->
	<a href="https://www.umich.edu/"><img src="/images/U-M_2color-HorizontalReversed.30px.png"></a>
	<!-- <p class="copyright">&#169; 2016 <a href="">The Mills Lab</a> powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> + <a href="http://mmistakes.github.io/skinny-bones-jekyll/" rel="nofollow">Skinny Bones</a>.</p> -->
</footer>

    </div>

    <script src="/js/vendor/jquery-1.9.1.min.js"></script>
    <script src="/js/main.js"></script>

  </body>

</html>
